Source/BestProjectEver/Private/Character/BPE_PlayerCharacter.cpp

```diff
@@ -369,6 +407,15 @@ void ABPE_PlayerCharacter::SetOverlappingWeapon(ABPE_Weapon* Weapon)
 	OnSetOverlappingWeapon(LastOverlappingWeapon);	
 }
 
+//----------------------------------------------------------------------------------------------------------------------
+void ABPE_PlayerCharacter::SetOverlappingSpawnPad(ABPE_SpawnPad* SpawnPad)
+{
+	ABPE_SpawnPad* LastOverlappingSpawnPad = OverlappingSpawnPad;
```

Comment:
Maybe this LastOverlappingSpawnPad could be const?

Reply:
![image](https://user-images.githubusercontent.com/85327912/173455754-334831e5-9293-4cff-b275-2c3f696b5b33.png)

![image](https://user-images.githubusercontent.com/85327912/173455409-09ce66c8-4763-45ea-a8f0-d14ebc2bee55.png)
Nou because here I set the widget visibility of that SpawnPad